### DISCUSSION TITLE
IgnoreColumns plugin functional?

### DISCUSSION THREAD

**Discussion by PlexPro** [WinMerge/winmerge#679] [Q&A] [answered]
I am trying to restrict a compare to ignore the first 24 columns in a row.

At "Plugins/Prediffer" I have IgnoreColumns.dll selected.  At "Plugins/Plugin Settings", I have IgnoreColumns.dll enabled, and the column ranges to ignore is set at 1-24

However, when I do a compare, it does not ignore these columns.  As these consist of data and time information (this is a log file), on many rows these 24 columns are the ONLY columns that differ.  So every row is selected as having differences.

Is there something that I be doing wrong, or does this function not work?

René V.

**Comment by sdottaka** [comment, marked as answer]
Select Plugins → Prediffer → IgnoreColumns.dll menu item.

![image](https://user-images.githubusercontent.com/98126/111052264-dcda7100-849c-11eb-9a36-9ab54c4eff16.png)

Alternatively, add the extension for your log file in the Plugins dialog window and click Plugins → Automatic Prediffer menu item to reopen the file.

![image](https://user-images.githubusercontent.com/98126/111052508-9dad1f80-849e-11eb-9602-c719b8cf9873.png)

![image](https://user-images.githubusercontent.com/98126/111052302-03001100-849d-11eb-9a39-96886c10cf9e.png)
